server/controllers/userController.js

```diff
@@ -0,0 +1,46 @@
+import { isBoolean } from "util";
+import database from "../models/index";
+import { validateData } from "../validations/validateData";
+import { signInSchema } from "../validationSchemas/user";
+
+import {
+  comparePassword,
+  generateJWToken,
+  httpResponse
+} from "../helpers/index";
+
+/**
+ * @description Generate login access token for a user
+ *
+ * @param {Request}  req
+ * @param {httpResponse} res
+ * @param {function} next
+ * @returns {object} HttpResponse {statusCode:int, success:boolean, data:object, message:string}
+ */
+export const userLogin = async (req, res, next) => {
+  const { email, password } = req.body;
+  const validated = await validateData({ email, password }, signInSchema);
+
+  if (isBoolean(validated)) {
```

Comment:
I told you what to do with this, please use the loadash .isEmpty to check whether this object is empty. 

```import isEmpty from 'lodash/isEmpty'

isEmpty(validated)

```